resources/data/elephpants.json

```diff
@@ -519,6 +519,14 @@
             "sponsor": "Mollie",
             "year": 2021,
             "image": "https://i.imgur.com/NScLVf1.jpg"
+        },
+        {
+            "id": 66,
+            "name": "KaraFun",
+            "description": "KaraFun",
+            "sponsor": "KaraFun",
+            "year": 2022,
+            "image": null
```

Comment:
Wouldn't it be nice to have an image for it?

Reply:
It would, but I'm not aware of any images that have been granted usage rights for this project

Reply:
Hello,
I work at KaraFun (the sponsor of this specimen). We have taken a photo of him and upload it at this URL: https://i.imgur.com/0TuGGoj.jpeg
We are ok for distribute photo under MIT Licence.
We would be happy to have our specimen present on elephpant.me

Reply:
@ppelisset Thank you for the image!  I have updated the PR